lib/resources/r4/related_person.yaml

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Im pretty sure at least AdministrativeGender is missing. Might want to review this: https://jira3.cerner.com/browse/ONTOLOGY-22653 to find any others

Reply:
add in remaining 
https://github.com/cerner/fhir.cerner.com/pull/254/commits/0118cf6b491931ce697425e63a3336710495db46#diff-5f74239458ffdc2051522276617b5f75R30

Reply:
OK So per the discussion the other day. I was wrong on how much we have to document. You can delete lines 30-123

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/53300d0b6211f4b0c77925c1e5ab9b78e4010aff#diff-5f74239458ffdc2051522276617b5f75L30

Reply:
+1